extensions/amp-google-assistant-assistjs/0.1/amp-google-assistant-voice-bar.js

```diff
@@ -53,7 +57,7 @@ export class AmpGoogleAssistantVoiceBar extends AMP.BaseElement {
     this.configService_.getWidgetIframeUrl('voicebar').then((iframeUrl) => {
       addAttributesToElement(iframe, {
         src: iframeUrl,
-        sandbox: 'allow-scripts',
+        sandbox: 'allow-scripts allow-same-origin'
```

Comment:
Can you explain this change? Given that the config is specified via JSON and the config allow you to override the Origin of this iframe's src, we've just opened up an XSS vulnerability.

When do we expect these to be the same origin? Is it only for testing? If so, we should assert that we're in test mode.

Reply:
Yes you are right. This was only for local testing. Removed.